bitnami/airflow/templates/deployment-web.yaml

```
@@ -185,6 +185,32 @@ spec:
               {{- else }}
               value: "no"
               {{- end }}
+              {{- if .Values.ldap.enabled }}
+            - name: AIRFLOW_LDAP_ENABLE
+              value: yes
```

I think we will need to quote this, or the parameter may be interpreted as a boolean in the YAML template

```suggestion
              value: "yes"
```